CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 All notable changes to this project will be documented in this file.
 
+## [0.3.0] - 2024-03-21
```

Comment:
Do I need to update this number? Also couldn't find the version txt so I just used the same number as the previous change log

Reply:
resolved